examples/streamed-list-objects/src/main/java/dev/openfga/sdk/example/StreamedListObjectsExample.java

```diff
@@ -0,0 +1,194 @@
+package dev.openfga.sdk.example;
+
+import dev.openfga.sdk.api.client.OpenFgaClient;
+import dev.openfga.sdk.api.client.model.ClientListObjectsRequest;
+import dev.openfga.sdk.api.client.model.ClientStreamedListObjectsOptions;
+import dev.openfga.sdk.api.client.model.ClientTupleKey;
+import dev.openfga.sdk.api.client.model.ClientWriteRequest;
+import dev.openfga.sdk.api.configuration.ClientConfiguration;
+import dev.openfga.sdk.api.configuration.Credentials;
+import dev.openfga.sdk.api.model.ConsistencyPreference;
+import dev.openfga.sdk.api.model.CreateStoreRequest;
+import dev.openfga.sdk.api.model.WriteAuthorizationModelRequest;
+import dev.openfga.sdk.errors.FgaInvalidParameterException;
+import java.util.ArrayList;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class StreamedListObjectsExample {
+    public static void main(String[] args) {
+        try {
+            new StreamedListObjectsExample().run();
+        } catch (Exception ex) {
+            // Avoid logging sensitive data; only display generic info
+            if (ex instanceof FgaInvalidParameterException) {
+                System.err.println("Validation error in configuration. Please check your configuration for errors.");
+            } else if (ex.getMessage() != null && ex.getMessage().contains("Connection refused")
+                    || (ex.getCause() != null
+                            && ex.getCause().getMessage() != null
+                            && ex.getCause().getMessage().contains("Connection refused"))) {
+                System.err.println(
+                        "Is OpenFGA server running? Check FGA_API_URL environment variable or default http://localhost:8080");
```

Comment:
If the default is "http://localhost:8080" can we reference it from a config? So that it is easier to maintain?